implementations/rust/build.gradle

```diff
@@ -13,6 +15,16 @@ ockamDir.eachFile(groovy.io.FileType.DIRECTORIES) {
 
 crates << new File('../../examples/rust/get_started')
 
+crates.each {
+  // Create and copy deny.toml to crates.
+  File src = new File(projectDir.toPath().toString() + '/deny.toml')
+  File dst = new File(projectDir.toPath().toString() + '/' + it.toString() + '/deny.toml')
+
+  if (dst.createNewFile()) {
+    dst.text = src.text
+  }
+}
+
```

Comment:
Instead of copying the `deny.toml`, could we us soft links?
In each crate folder run `ln -s ../../deny.toml deny.toml` (the current DEVELOP.md file is linked like this)

If cargo deny picks up such a linked file, we won't have to copy it / clean it / gitignore it etc.

Reply:
Thanks for this. Made the change.